packages/config-helper/project.json

```diff
@@ -32,14 +32,15 @@
 		"bump-version": {
 			"executor": "@jscutlery/semver:version",
 			"options": {
-				"versionTagPrefix": "${target}@"
+				"versionTagPrefix": "${target}@",
+				"postTargets": ["config-helper:publish-npm"]
 			}
 		},
 		"publish-npm": {
 			"executor": "ngx-deploy-npm:deploy",
 			"options": {
 				"access": "public",
-				"no-build": true
+				"noBuild": true
```

Comment:
I made a test using `dryRun` feature of ngx-deploy-npm, and the option `no-build` was not recognized; the library's build is still taking place.
That's the reason for this change.